experimental/gf-recalculate-of-image-wrapper-click.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Gravity Perks // Gravity Forms //  Recalculate on checkbox change
+ * https://gravitywiz.com/
+ *
+ * Instructions:
```

Comment:
```suggestion
 * This snippet is needed to work around a bug in Gravity Forms 2.9 where calculations are not
 * re-processed when the actual image of an Image Choice is clicked. It works fine if you click on
 * the label or the checkbox/radio itself.
 *
 * Instructions:
```

Ensure there's a single empty line between `https://gravitywiz.com` and this description block